spec.html

```diff
@@ -30263,18 +30266,21 @@ <h1>
               1. Let _parameterSym_ be the grammar symbol |FormalParameters[+Yield, +Await]|.
               1. Let _fallbackProto_ be *"%AsyncGeneratorFunction.prototype%"*.
             1. Let _argCount_ be the number of elements in _parameterArgs_.
+            1. Let _parametersStrings_ be a new empty List.
```

Comment:
Pluralization is inconsistent between _parameterArgs_ and _parametersStrings_.
```suggestion
            1. Let _parameterStrings_ be a new empty List.
```
(And similarly in the HostEnsureCanCompileStrings parameter.)